### ISSUE TITLE
Failed to install buckaroo on mac with homebrew

### ISSUE THREAD

**Issue by lovely-error** [LoopPerfect/buckaroo#376]
Hello. I tried to install the utility with this command (`brew install loopperfect/lp/buckaroo`) from getting started section, but the installation failed with these errors:
```
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/loopperfect/homebrew-lp/buckaroo@0.1.2.rb
buckaroo@0.1.2: Calling depends_on :java is disabled! Use "depends_on "openjdk@11", "depends_on "openjdk@8" or "depends_on "openjdk" instead.

Error: Cannot tap loopperfect/lp: invalid syntax in tap!
```

I have no idea what to do, please help.

This is brew version:
```
Homebrew 2.7.0
Homebrew/homebrew-core (git revision 47333; last commit 2020-12-23)
Homebrew/homebrew-cask (git revision 312569; last commit 2020-12-24)
```
And macos is 10.15.6

**Comment by nikhedonia** [comment]
Thanks for reaching out!

I think homebrew is trying to tell us that this recipe is outdated.
Todays buckaroo doesn't depend on java anymore and we provide self-containing binaries that can be found on our [release page](https://github.com/LoopPerfect/buckaroo/releases)

**Comment by njlr** [comment]
`0.1.2` is a very old version; the latest version is `2.2.0`.

**Comment by lovely-error** [comment]
I tried to get buckaroo from brew, and put the command in it - `brew install loopperfect/lp/buckaroo` - yet there are errors.
There are bunch of them but the one that shows the most recent version is `Error: Invalid formula: /usr/local/Homebrew/Library/Taps/loopperfect/homebrew-lp/buckaroo@1.2.1.rb`. The rest of them look like this one, but version numbers are lower.
As @njlr just said, the most recent version is 3 and apparently, brew doesn't see it.

I assume that on release there are binaries and macos version is among them, yet it is not recognized as executable, but as an unknown file on my pc. Could you clarify this bit, please, @nikhedonia?

Anyway, is there a reason why I cannot install with brew? Is something wrong happening on my side?

**Comment by njlr** [comment]
I have added `3.0.2` to the Homebrew versions. I do not have a Mac to hand for testing, but it works in Linuxbrew.

**Comment by nikhedonia** [comment]
> I assume that on release page there are binaries and macos version is among them, yet it is not recognized as executable, but as an unknown file on my pc. Could you clarify this bit, please, @nikhedonia?

I just got a hold on a mac with big sur installed. 
It works provided you explicitly grant permissions in Security & Privacy settings.
So the first run in iTerm will open a dialog box and fail execution of buckaroo:

```
chmod +x buckaroo
./buckaroo // opens dialog box
```

Then buckaroo will show up in the "Security & Privacy" settings and once I granted permissions it worked.

**Comment by SamuelMarks** [comment]
Hey guys just gave it a shot, adding the latest version to `brew` https://github.com/Homebrew/homebrew-cask/pull/103094

**Comment by jayvdb** [comment]
> Hey guys just gave it a shot, adding the latest version to brew Homebrew/homebrew-cask#103094

That was rejected ;-(, saying it should be instead submitted to https://github.com/Homebrew/homebrew-core

**Comment by SamuelMarks** [comment]
Yeah at this point I cbf. I was on a tirade with all the different package managers to try and find one which worked out-of-the-box for cross-platform GUI dev: https://github.com/microsoft/vcpkg/issues/17194

**Comment by lovelace** [comment]
This apparently has been an issue for well over a year, and yet nothing has been done.  While looking around for a C++ library package dependency manager for software development on the mac, I found reference to buckaroo, but since trying to use the homebrew installation instructions within your very own documentation resulted in numerous errors I have decided not to pursue use of your solution.  If in the future you actually care about fixing things, I might reconsider.  Until then, since you can't be bothered to support the mac correctly, you shouldn't expect people on the mac to use your software.

For reference, this is the error message I got when trying to install:
```
> brew install loopperfect/lp/buckaroo
==> Tapping loopperfect/lp
Cloning into '/usr/local/Homebrew/Library/Taps/loopperfect/homebrew-lp'...
remote: Enumerating objects: 200, done.
remote: Total 200 (delta 0), reused 0 (delta 0), pack-reused 200
Receiving objects: 100% (200/200), 26.16 KiB | 1.74 MiB/s, done.
Resolving deltas: 100% (87/87), done.
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/loopperfect/homebrew-lp/buckaroo@0.1.2.rb
buckaroo@0.1.2: Unsupported special dependency :java
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/loopperfect/homebrew-lp/buckaroo@0.1.1.rb
buckaroo@0.1.1: Unsupported special dependency :java
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/loopperfect/homebrew-lp/buckaroo@1.4.0.rb
buckaroo@1.4.0: Unsupported special dependency :java
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/loopperfect/homebrew-lp/buckaroo@0.1.0.rb
buckaroo@0.1.0: Unsupported special dependency :java
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/loopperfect/homebrew-lp/buckaroo@1.5.0.rb
buckaroo@1.5.0: Unsupported special dependency :java
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/loopperfect/homebrew-lp/buckaroo@1.4.1.rb
buckaroo@1.4.1: Unsupported special dependency :java
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/loopperfect/homebrew-lp/buckaroo@1.3.1.rb
buckaroo@1.3.1: Unsupported special dependency :java
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/loopperfect/homebrew-lp/buckaroo@1.2.0.rb
buckaroo@1.2.0: Unsupported special dependency :java
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/loopperfect/homebrew-lp/buckaroo@1.3.0.rb
buckaroo@1.3.0: Unsupported special dependency :java
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/loopperfect/homebrew-lp/buckaroo@1.2.1.rb
buckaroo@1.2.1: Unsupported special dependency :java
Error: Cannot tap loopperfect/lp: invalid syntax in tap!
```
